EnglishFilter/sections/general_extensions.txt

```diff
@@ -232,6 +232,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75284
+curimovie.link#%#//scriptlet("prevent-window-open", "1", "errcarsvex.casa")
```

Comment:
Why did you use this scriptlet? For app and extension this `mp4cage.com#%#//scriptlet("abort-on-property-read", "BetterJsPop")` is enough on my end.

Reply:
@zloyden Please try like this, disable AdGuard AdBlocker Extension, enable AdGuard for Windows. Go to that website click anywhere except the player. You must see pop-ups.

Reply:
I've also checked it with app and US IP. Not reproduced, only popup from the player. @AdamWr please take a look on your end.

Reply:
@zloyden Did you enable the AdGuard AdBlocker Extension? If so, of course the pop-ups will be blocked.

Reply:
Nope, only the app with/without US IP.

Reply:
@zloyden By the way, I can't reproduce it with US IP. Please try with Belgium, Singapore and Sweden.

Reply:
> Belgium, Singapore and Sweden.

Unfortunately, I still have the same. Let's wait for my colleague and his review.

Reply:
@zloyden Let me send you a screenshot.

Reply:
@zloyden 
Press the blue thing in this screenshot.
![image](https://user-images.githubusercontent.com/76908410/108588460-48e12200-7394-11eb-90d6-7e3e4d6b2caf.png)

Reply:
Don't have it even without AdGuard, with/without VPN.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/108588797-21765f00-736c-11eb-900c-01b9ff71f186.png)

</details><br/>

Reply:
Please use the app.

Reply:
It appears sporadically.

Reply:
And wait for sometime.

Reply:
@zloyden Please use the app and it appears sporadically. Also, wait for sometime in the site.

Reply:
I've tested it in different cases (with app too) and still the same. Btw, is this their element or a third-party one? If a third-party is created by what? Show a screenshot with devtools opened.

Reply:
@zloyden It's impossible to inspect as the blue thing goes out. I will element pick it, wait up.

Reply:
@zloyden It also eats my AdGuard Assistant up.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/76908410/108589521-9ad87680-7399-11eb-921e-4274947ded79.png)

</details><br/>

Reply:
Hm, strange.


> AdGuard Assistant

It's obsolete now. You should use this one https://adguard.com/en/adguard-assistant/overview.html#chrome
What version of our app have you used? Is it release?
Also please test this instead of a scriptlet `||errcarsvex.casa^$third-party`

Reply:
@zloyden No use. I use the latest nightly with only AdGuard Filters enabled. They changed their code. I think I found something.

Reply:
@zloyden Fixed. Please take a look in a while.

Reply:
> I use the latest nightly

That's good to hear. That's how it should be.

Reply:
Why?